src/senaite/sync/browser/templates/sync.pt

```diff
@@ -122,6 +122,28 @@
           </div>
         </div>
 
+        <!-- Content Types -->
+        <div class="field form-group field">
+          <label i18n:translate=""
+                 class="form-control-label"
+                 for="content_types">
+            Content Types
+            <span i18n:translate=""
+                  class="help formHelp">
+              If filled, only indicated Content Types (separated by comma) will be imported. E.g: 'Sample, Client, Worksheet'
+            </span>
+          </label>
+          <div class="form-group input-group">
+            <span class="input-group-addon"><i class="glyphicon glyphicon-lock"></i></span>
+            <input type="text"
+                   size="45"
+                   class="form-control"
+                   id="content_types"
+                   tal:attributes="value view/content_types|nothing"
+                   name="content_types"/>
```

Comment:
I would render here a multi checkbox field instead of requiring the user to type in the content types by hand.
Note: You also don't handle eventual spaces in between in your code, e.g. `   Sample,  Worksheed   , FooType`

Reply:
You are right that using checkboxes is a way better idea. But we discussed it with @xispa how to generate that list of checkboxes.
Should it be _hardcoded_?! - Not a good solution plus what about different content types of each add on?! Should we retrieve them from Portal Types registry? Then the list is too long...
So we ended up with letting the user enter them considering that the person who manages `sync`, will have some acknowledge.

And spaces are removed here: https://github.com/senaite/senaite.sync/pull/16/files#diff-32423e22290aaa3db2cdebe0c85fa752R106

Reply:
Ok got it. Then maybe you should do something like this afterwards:
```python
portal_types = api.get_tool("portal_types")
content_types = filter(lambda ct: ct in portal_types, content_types)
```
Just a scribble, so you need to find out how to retrieve the current registered types...

Reply:
That is a good idea, I added it to form submission in the view.